sites/platform/src/development/variables/use-variables.md

```diff
@@ -348,6 +348,7 @@ and at runtime.
 | `{{< vendor/prefix >}}_SOURCE_DIR`       | Yes   | No      | The path to the root directory of your code repository in the context of a running [source operation](../../create-apps/source-operations.md). The directory contains a writable copy of your repository that you can commit to during the operation. |
 | `{{< vendor/prefix >}}_TREE_ID`          | Yes   | Yes     | The ID of the tree the application was built from, essentially the SHA hash of the tree in Git. Use when you need a unique ID for each build. |
 | `{{< vendor/prefix >}}_VARIABLES`        | Some  | Some    | A base64-encoded JSON object with all user-defined project and environment variables that don't use a [prefix](./_index.md#variable-prefixes). The keys are the variable names and the values are the variable values. Availability during builds and at runtime depends on the settings for each variable. See how to [access individual variables](#access-variables-in-a-shell). |
+| `{{< vendor/prefix >}}_VENDOR`        | Yes  | No    |  | Allows you to change the behavior of the build according to the vendor. |
```

Comment:
We could specify what vendor means (upsun or platformsh). (Ignore that comment if it's a name we use in other places.)